plugins/faustwp/includes/blocks/functions.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ * Functions related to block support.
+ *
+ * @package FaustWP
+ */
+
+namespace WPE\FaustWP\Blocks;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Returns the contents of theme.json if present.
+ *
+ * @return object The contents of theme.json.
+ */
+function get_theme_json() {
+	$file_path = get_template_directory() . '/theme.json';
```

Comment:
I believe we want to use the [processed theme.json data](https://developer.wordpress.org/reference/classes/wp_theme_json_resolver/), not just the default values in `theme.json`.

Reply:
Thanks @justlevine that makes sense, I appreciate the feedback!

Would you say that [WP_Theme_JSON_Resolver::get_merged_data](https://developer.wordpress.org/reference/classes/wp_theme_json_resolver/get_merged_data/) would be the appropriate method in this case?

Reply:
Sure, if it makes sense to roll our own caching. Otherwise, probably [wp_get_global_settings()](https://developer.wordpress.org/reference/functions/wp_get_global_settings/).